src/components/Modal/_Modal.scss

```diff
@@ -1,6 +1,7 @@
 .no-scroll {
     position: fixed;
     top: -100vh;
```

Comment:
@scooooooooby  what was the idea behind this rule?

Reply:
I can't remember, I might not have written this (despite possibly showing up in the `git blame`). @hokei, do you remember?